File: Group 57/level 22/Classwork/cw3.py
# 3) მომხმარებელს შემოატანინეთ რიცხვი, შემდეგ კი დაბეჭდეთ ერთიდან ამ რიცხვამდე მხოლოდ ლუწი რიცხვების ჯამი ცალკე და მხოლოდ კენტი რიცვხების ჯამი ცალკე, გამოიყენეთ for loop.

num = int(input("Enter Your Number: "))

sum = 0

sum2 = 0

for i in range(num):
    if i % 2 == 0:
        sum += i
    if i % 2 != 0:
        sum2 += i

print(sum)
print(sum2)

File: Group 57/level 101/Classwork/cw3.js
//3)მომხმარებელს შეეკითხეთ რიცხვი,შემდეგ 1 დან მომხმ შემოტანილ რიცხვამდე დაბეჭდეთ ყველა რიცხვი 

let userNumber = prompt("Enter a Number:");

userNumber = Number(userNumber);

let i = 1; 

while (i <= userNumber) {
    console.log(i); 
    i++;
}